src/cli/self_update.rs

```diff
@@ -361,7 +361,7 @@ macro_rules! pre_install_msg_win {
     () => {
         pre_install_msg_template!(
             "This path will then be added to your `PATH` environment variable by
-modifying the `HKEY_CURRENT_USER/Environment/PATH` registry key."
+modifying the `PATH` registry key at `HKEY_CURRENT_USER\\Environment`."
```

Comment:
Nit: Maybe use a raw string here so that the `\` looks clearer?

Reply:
changed